runconfig/hostconfig_windows.go

```diff
@@ -10,25 +10,22 @@ import (
 // DefaultDaemonNetworkMode returns the default network stack the daemon should
 // use.
 func DefaultDaemonNetworkMode() container.NetworkMode {
-	return container.NetworkMode("default")
+	return container.NetworkMode("NAT")
```

Comment:
I don't think there will be any issue. The driver name is uppercase but the network name will be lowercase